components/search/index.css

```
@@ -41,43 +193,171 @@ governing permissions and limitations under the License.
     -webkit-appearance: none;
   }
 
-  border-radius: var(--spectrum-alias-search-border-radius);
+  block-size: var(--mod-search-block-size, var(--spectrum-search-block-size));
+  padding-block-start: calc(
+    var(--mod-search-top-to-text, var(--spectrum-search-top-to-text)) - 
+    var(--mod-search-border-width, var(--spectrum-search-border-width))
+  );
+  padding-block-end: calc(
+    var(--mod-search-bottom-to-text, var(--spectrum-search-bottom-to-text)) - 
+    var(--mod-search-border-width, var(--spectrum-search-border-width))
+  );
+
+  border-radius: var(--mod-search-border-radius, var(--spectrum-search-border-radius));
+  border-width: var(--mod-search-border-width, var(--spectrum-search-border-width));
+
+  font-family: var(--mod-search-font-family, var(--spectrum-search-font-family));
+  font-weight: var(--mod-search-font-weight, var(--spectrum-search-font-weight));
+  font-style: var(--mod-search-font-style, var(--spectrum-search-font-style));
+  line-height: var(--mod-search-line-height, var(--spectrum-search-line-height));
+
+  background-color: var(--highcontrast-search-background-color, var(--mod-search-background-color, var(--spectrum-search-background-color)));
+  border-color: var(--highcontrast-search-border-color-default, var(--mod-search-border-color-default, var(--spectrum-search-border-color-default)));
+  color: var(--highcontrast-search-color-default, var(--mod-search-color-default, var(--spectrum-search-color-default)));
+
+  &::placeholder {
+    color: var(--highcontrast-search-color-default, var(--mod-search-color-default, var(--spectrum-search-color-default)));
+  }
+
+  /* Hover */
+  .spectrum-Search-textfield:hover &,
+  &:hover {
+    color: var(--highcontrast-search-color-hover, var(--mod-search-color-hover, var(--spectrum-search-color-hover)));
+    border-color: var(--highcontrast-search-border-color-hover, var(--mod-search-border-color-hover, var(--spectrum-search-border-color-hover)));
```

```suggestion
  &:hover {
    --spectrum-search-color: var(--highcontrast-search-color-hover, var(--mod-search-color-hover, var(--spectrum-search-color-hover)));
    --spectrum-search-border-color: var(--highcontrast-search-border-color-hover, var(--mod-search-border-color-hover, var(--spectrum-search-border-color-hover)));
```

^ this might be another great place for state-based variable override

---

I don't think I'd be able to do that here with all of these pseudos for the textfield input, because each one is overriding the color and border-color being set on the textfield subcomponent (unless I added some extra specificity to the selector that would set the `color:` to the local variable)

---

This whole section has been replaced with custom property passthroughs 🎉